core/unit_test/TestTeamTeamSize.hpp

```diff
@@ -121,12 +121,21 @@ void test_team_policy_max_recommended_static_size(int scratch_size) {
                            .set_scratch_size(0, Kokkos::PerTeam(scratch_size)),
                        FunctorFor<T, N, PolicyType, S>());
   MyArray<T, N> val;
+  double n_leagues = 10000;
+  // FIXME_HIP
```

Comment:
Any idea what the issue here is?

Reply:
That's the segfault that we reported to AMD and we talked about during the last group meeting